scipy/fft/tests/test_numpy.py

```diff
@@ -279,19 +279,19 @@ def worker(args, q):
                 'Function returned wrong value in multithreaded context')
 
     def test_fft(self):
-        a = np.ones(self.input_shape) * 1+0j
+        a = np.full(self.input_shape, 1+0j)
```

Comment:
Nit: these are all still filling with ones.
```suggestion
        a = np.ones(self.input_shape, dtype=np.complex128)
```